user-manual/metadata/METS.rst

```diff
@@ -0,0 +1,182 @@
+* dmdSec (descriptive metadata section) (1 or more)
+* fileSec (file section) (1 only)
+* structMap (structural map) (1 or more)
+
+.. literalinclude:: scripts/metsfile.xml
+   :language: bash
+   :lines: 1-26
+
+Below is a more detailed outline of the above sections present in AIPs created
+in Archivematica.
+
+Skip to:
+
+* :ref:`<metsHdr> <metshdr>`
+* :ref:`<dmdSec> <dmdsec>`
+* :ref:`<amdSec> <amdsec>`
+* :ref:`<fileSec> <filesec>`
+* :ref:`<structMap> <structmap>`
+
+
+.. _metshdr:
+
+<metsHdr>
+^^^^^^^^^
+
+There will be one METS header section for each METS file. It will contain a
+CREATEDATE attribute.
+
+.. code:: xml
+
+  <metsHdr CREATEDATE=”2019-06-18T23:52:11”/>
+
+If the AIP has been reingested, the metsHder section will also contain a
+LASTMODDATE.
+
+.. code:: xml
+
+  <metsHdr CREATEDATE=”2019-06-18T23:52:11” LASTMODDATE=”2019-06-27T00:28:56”/>
+
+
+.. _dmdsec:
+
+<dmdSec>
+^^^^^^^^
+
+There will be one dmdSec for the whole AIP, which contains PREMIS Intellectual
+Entity information.
+
+.. literalinclude:: scripts/METS.xml
+   :language: xml
+   :lines: 15-35
+
+There may also be one or more dmdSecs for descriptive metadata. The dmdSecs are
+numbered dmdSec_1, dmdSec_2, etc. and contain Dublin Core metadata as a default.
+
+.. literalinclude:: scripts/METS.xml
+   :language: xml
+   :lines: 64-91
+
+MDTYPE will be **DC** for Dublin Core metadata; if another metadata standard is
+included then the MDTYPE will be **OTHER**. 
+
+If the user does not enter any DC metadata during transfer/ingest and no DC
+metadata was included with the transfer (e.g. as a metadata.csv file, see
+:ref:`Import metadata <import-metadata>`), there will be no additional dmdSecs.
+
+
+.. _amdsec:
+
+<amdSec>
+^^^^^^^^^
+
+There is one amdSec for each object and wraps PREMIS entities as follows:
+
+* PREMIS Object: techMD (tehnical metadata)
+* PREMIS Event: digiprovMD (digital provenance metadata)
+* PREMIS Agent: digiprovMD (digital provenance metadata)
+* PREMIS Rights: rightsMD (rights metadata)
+
+Each amdSec will include one techMD. The techMD is used to wrap PREMIS Object
+semantic units, such as objectIdentifier, objectCharacteristics and
+originalName. See the example techMD below:
+
+.. literalinclude:: scripts/METS.xml
+   :language: xml
+   :lines: 212-258
+
+Each amdsec will include multiple digiprovMDs for PREMIS events related to the
+PREMIS Object. Below is an example of a PREMIS event for a fixity check:
+
+   .. literalinclude:: scripts/METS.xml
+      :language: xml
+      :lines: 820-854
+
+An amdSec for an original object may also contain one or more rightsMDs. The
+rightsMD may contain a reference to metadata in another file, such as rights.csv
+uploaded with a transfer. Below is an example of a PREMIS Rights statement in
+rightsMD section of the AIP METS file:
+
+.. literalinclude:: scripts/METS.xml
+   :language: xml
+   :lines: 2073-2108
+
+An amdSec will also have one ore more sourceMD sections. This is the only
```

Comment:
```suggestion
An amdSec will also have one or more sourceMD sections. This is the only
```

Reading below, should it `zero or more` where it looks like it isn't necessarily used for a standard transfer? Also, shall we expand on `sourceMD` to be `source metadata` as in the above approach?

Reply:
Good point, will clarify and add your suggestion.